chart/permissions-api/templates/deployment.yaml

```diff
@@ -94,11 +94,11 @@ spec:
               protocol: TCP
           livenessProbe:
             httpGet:
-              path: /healthz
+              path: /livez
               port: http
           readinessProbe:
             httpGet:
-              path: /healthz
+              path: /readyz
```

Comment:
Let's move these to a separate PR.

Reply:
This change is part of the echox change. echox provides the readyz endpoint, so I feel this should be part of the same pr.

Reply:
ah, I didn't know it was part of that, makes sense. Let's keep it here.